client/reader/sidebar/reader-sidebar-network/index.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { localize } from 'i18n-calypso';
+import { identity } from 'lodash';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import ExpandableSidebarMenu from 'layout/sidebar/expandable';
+import ReaderSidebarNetworkList from './list';
+import getReaderFollowsP2 from 'state/reader/follows/selectors/get-reader-follows-p2';
+
+export class ReaderSidebarNetwork extends Component {
+	static propTypes = {
+		path: PropTypes.string.isRequired,
+		unseen: PropTypes.object,
+		isOpen: PropTypes.bool,
+		onClick: PropTypes.func,
+		translate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		translate: identity,
+	};
+
+	render() {
+		const { isOpen, translate, onClick, unseen } = this.props;
+		return (
+			<ul className={ unseen && unseen.status ? 'has-unseen' : '' }>
+				{ /* eslint-disable-next-line wpcalypso/jsx-classname-namespace */ }
+				{ unseen && unseen.status && <span className="sidebar__bubble" /> }
+				<ExpandableSidebarMenu
+					expanded={ isOpen }
+					title={ translate( 'P2' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **6** times:
`translate( 'P2s' )` *ES Score: 7*